server/Arcadia.Assistant.CSP/SickLeaves/CspSickLeavesRegistry.cs

```diff
@@ -99,50 +94,6 @@ protected override void OnReceive(object message)
                     this.ScheduleNextDatabaseRefresh();
                     break;
 
-                case RefreshDatabaseCreatedSuccess msg:
-                    this.logger.Debug($"Sick leave with id {msg.Event.EventId} for employee {msg.Event.EmployeeId} " +
-                        "was added to CSP database manually");
-
-                    Context.System.EventStream.Publish(new CalendarEventCreated(msg.Event, msg.CreatedBy, msg.Timestamp));
```

Comment:
we don't need to publish Created event anymore?

Reply:
It is published in the line 187.